.github/workflows/tests.yml

```
@@ -413,12 +413,15 @@ jobs:
   # Threshold: p95 < 2s, error rate < 5%.
   # Catches per-PR latency regressions before merge.
   # ─────────────────────────────────────────────────────────────────────────
-  k6-smoke:
-    name: k6 smoke (30s)
-    runs-on: ubuntu-latest
-    needs: [self-heal]
-    if: always() && !cancelled() && github.event_name == 'pull_request'
-    timeout-minutes: 5
+    k6-smoke:
```

actionlint (1.7.12)

[error] unexpected key "k6-smoke" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "snapshot", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)